Hardcore.lua

```diff
@@ -370,78 +376,83 @@ function Hardcore:PLAYER_DEAD()
 	SendChatMessage(messageString, "GUILD", nil, nil)
 
 	-- Send add command to addon for this death
-	local deathData = string.format("%s%s%s%s%s%s%s%s%s%s%s",
-									playerId,
-									COMM_FIELD_DELIM,
-									playerName,
-									COMM_FIELD_DELIM,
-									localizedClass,
-									COMM_FIELD_DELIM,
-									playerLevel,
-									COMM_FIELD_DELIM,
-									mapId,
-									COMM_FIELD_DELIM,
-									time())
-
-	local commMessage = COMM_COMMANDS[2]..COMM_COMMAND_DELIM..deathData
+	local deathData = string.format("%s%s%s%s%s%s%s%s%s%s%s", playerId, COMM_FIELD_DELIM, playerName, COMM_FIELD_DELIM,
+		localizedClass, COMM_FIELD_DELIM, playerLevel, COMM_FIELD_DELIM, mapId, COMM_FIELD_DELIM, time())
+
+	local commMessage = COMM_COMMANDS[2] .. COMM_COMMAND_DELIM .. deathData
 	if CTL then
 		CTL:SendAddonMessage("ALERT", COMM_NAME, commMessage, "GUILD")
 	end
 end
 
 function Hardcore:PLAYER_UNGHOST()
-	if Hardcore_Settings.enabled == false then return end
-	if UnitIsDeadOrGhost("player") == 1 then return end -- prevent message on ghost login or zone
+	if Hardcore_Settings.enabled == false then
+		return
+	end
+	if UnitIsDeadOrGhost("player") == 1 then
+		return
+	end -- prevent message on ghost login or zone
 
 	local playerName, _ = UnitName("player")
-	local message = playerName.." has resurrected!"
+	local message = playerName .. " has resurrected!"
 	SendChatMessage(message, "GUILD", nil, nil)
 
-	-- Screen notification
-	Hardcore_Notification_Text:SetText(message)
-	Hardcore_Notification_Frame:Show()
-	PlaySound(8959)
-	C_Timer.After(COMM_DELAY, function()
-		Hardcore_Notification_Frame:Hide()
-	end)
+	Hardcore:ShowAlertFrame(ALERT_STYLES.spirithealer, message)
+
+    -- Screen notification
+    -- Hardcore_Notification_Text:SetText(message)
+    -- Hardcore_Notification_Frame:Show()
+    -- PlaySound(8959)
+    -- C_Timer.After(COMM_DELAY, function()
+    --     Hardcore_Notification_Frame:Hide()
+    -- end)
```

Comment:
Can we remove this?

Reply:
Yep, it's pure confidence-code - I'd prefer to remove after testing, but w/e works for you